src/style.css

```diff
@@ -0,0 +1,131 @@
+.page {
```

Comment:
it`s better to create a separate file for each BEM block

Reply:
pay attention to comments, remove this entire file